Gemfile

```diff
@@ -1,10 +1,14 @@
 source 'https://rubygems.org'
 
 gem 'panoptes-client', '~> 0.3.7'
-gem 'pry'
 gem 'puma'
 gem 'sinatra'
 
-group :development, :test do
+group :development do
+  gem 'pry'
+end
+
+group :test do
+  gem 'rack-test'
   gem 'rspec'
```

Comment:
RSpec needs to remain in dev/test for the CLI to work

Reply:
i'm not sure i understand, i can run rspec just fine via on the command line

Reply:
hmm interesting, i thought different but tried it on this branch and you're right. sorry.